app/controllers/api/v1/memberships_controller.rb

```diff
@@ -18,7 +18,7 @@ def create
 
   private
 
-  def add_active_resources_scope
-    false
+  def policy_object
+    RoledControllerPolicy.new(api_user, resource_class, resource_name, action_name, params, add_active_resources_scope: false)
```

Comment:
did you want memoization here?

Reply:
Gets added in the `pundit-scopes` branch.